src/master-playlist-controller.js

```diff
@@ -1346,9 +1346,6 @@ export class MasterPlaylistController extends videojs.EventTarget {
     if (usingAudioLoader && unsupportedAudio && this.media().attributes.AUDIO) {
       const audioGroup = this.media().attributes.AUDIO;
 
-      this.mediaTypes_.AUDIO.activePlaylistLoader.pause();
```

Comment:
this is now handled by `mediachanging` in `media-groups.js` We already reset the main loader on `mediachanging`. This prevents bad appends from excluding the main playlist and resets the playback watcher sooner.

Reply:
I think this needs to be added back since `mediachanging` was removed